backend/src/test/java/com/bakdata/conquery/integration/IntegrationTests.java

```diff
@@ -160,13 +161,15 @@ private DynamicTest readTest(Resource resource, String name) {
 		try (InputStream in = resource.open()) {
 			JsonIntegrationTest test = new JsonIntegrationTest(in);
 
-			name = test.getTestSpec().getLabel();
+			String testLabel = Optional.ofNullable(test.getTestSpec().getLabel())
+									   // If no label was defined use the filename part before the first dot
+									   .orElse(name.split("\\.", 1)[0]);
 
 			return DynamicTest.dynamicTest(
-					name,
-					URI.create("classpath:/" + resource.getPath()),
+					testLabel,
+					resource.getURI(),
```

Comment:
dadurch kommt man jetzt endlich zur `*.test.json` wenn man in den Test-Ergebnissen auf den Test klickt